### PR TITLE
audit: enforce https for apache.org

### PR DESCRIPTION
**Pull request by vszakats**
- including all apache.org subdomains
- for `homepage` as well

### CODE REVIEW

**Comment by DomT4** [comment]
Wasn't there a problem with this last time we had the big audit discussion? Around some Apache domains not rendering at all correctly under SSL/TLS?

**Comment by vszakats** [comment]
Some pages exhibit "mixed-content" problems just like some do on `github.io` (notably the mirror selection page), but I haven't seen examples of _broken_ pages lately. Of course this doesn't mean there can't be any; for this reason the same rules may be applied to this domain as you have added for `github.io`: consider it a page bug and/or make an exception from the rule. Overall I interpret the `*.apache.org` wildcard certificate as a clear push for SSL/TLS on their whole domain.

File: Library/Homebrew/cmd/audit.rb [hunk 1]
```diff
@@ -300,6 +300,9 @@ def audit_urls
     if homepage =~ %r[^http://[^/]*github\.io/]
       problem "Github Pages links should be https:// (URL is #{homepage})."
     end
+    if homepage =~ %r[^http://[^/]*\.apache\.org/]
```
Comment (mikemcquaid) [inline Library/Homebrew/cmd/audit.rb]: Add a newline above here.

Comment (vszakats) [inline Library/Homebrew/cmd/audit.rb]: Added.

**Comment by DomT4** [comment]
> consider it a page bug and/or make an exception from the rule.

Solid point. Concerns withdrawn.

**Comment by mikemcquaid** [comment]
Thanks @vszakats!

**Pull request closed without merging**